docs/content/preview/migrate/yb-voyager/install-yb-voyager.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Install yb-voyager
+linkTitle: Install yb-voyager
+description: Steps to install yb-voyager.
+beta: /preview/faq/general/#what-is-the-definition-of-the-beta-feature-tag
+menu:
+  preview:
+    identifier: install-yb-voyager
+    parent: yb-voyager
+    weight: 103
+isTocNested: true
+showAsideToc: true
+---
+
+yb-voyager is a command line executable program that supports migrating databases from PostgreSQL, Oracle, and MySQL to a YugabyteDB database. yb-voyager keeps all of its migration state, including exported schema and data, in a local directory called the *export directory*. For more information, refer to [Export directory](../../yb-voyager/reference/#export-directory).
+
+The following sections include steps to install yb-voyager, and preparing the source and target databases.
+
+Follow the steps to install YB Voyager on a machine which satisfies the [Prerequisites](../../yb-voyager/prerequisites/).
+
+- Clone the yb-voyager repository.
+
+```sh
+git clone https://github.com/yugabyte/yb-voyager.git
+```
+
+- Change the directory to `yb-voyager/installer_scripts`.
+
+```sh
+cd yb-voyager/installer_scripts
+```
+
+- Depending on the Linux distribution (CentOS or Ubuntu) you're running, execute the appropriate installer script:
+
+```sh
+#CentOS
+./yb_voyager_installer__centos.sh
+```
+
+```sh
+#Ubuntu
+./yb_voyager_installer__ubuntu.sh
+```
+
```

Comment:
The two scripts are now unified into one `install-yb-voyager` script. This new script internally detects whether it is running on CentOS, Ubuntu, or MacOS.